packages/expo-audio/expo-module.config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "platforms": ["ios", "android"],
+  "platforms": ["ios", "tvos", "android"],
```

Comment:
```suggestion
  "platforms": ["apple", "android"],
```

Reply:
It does, but only when it's specified as supported in the podspec. The podspec should be the only source of truth to avoid confusion. `ios`, `tvos` and `macos` are all deprecated as we can use just one. It's one step less for people who add support for more platforms (not just us, other module authors too).